Shared/Features/Freaks/ViewModel/FreaksViewModel.swift

```diff
@@ -10,8 +11,16 @@ final class FreaksViewModel: ObservableObject {
         getFreaks()
     }
 
+    func loadFreaks() {
+        Network.shared.getAllFreaks { [weak self] result, _ in
+            guard let self = self else { return }
+            self.freaks = result
+            self.allFreaks = result
```

Comment:
Why do we need two variables to store all freaks?

Reply:
We need this for the filterFreaks() function.